apache/superset
# [dashboards] Fix, update dashboard owners not propagating to charts o…

### CATEGORY


### SUMMARY
Regression when updating a dashboard the dashboard's charts owners are not updated.
When changing the dashboard owners we expect that all the charts that belong to the dashboard to be updated with the same dashboard owners (and preserve their own also)

Note: Will add tests to cover this regression

### TEST PLAN
- Login with `admin`
- Create 2 users `gamma` and `alpha`
- Update a dashboard setting it's new owners with `gamma` and `alpha`
- Check that all the charts that belong to the dashboard get it's owners updated with `gamma`, `alpha` and `admin`

### ADDITIONAL INFORMATION

### REVIEWERS
@suddjian @willbarrett
